src/ape/api/providers.py

```diff
@@ -200,12 +200,13 @@ def get_balance(self, address: AddressType) -> int:
         """
 
     @abstractmethod
-    def get_code(self, address: AddressType) -> ContractCode:
+    def get_code(self, address: AddressType, **kwargs) -> ContractCode:
```

Comment:
so i made sure, we are allowed to add `**kwargs` to abstract methods without it breaking the plugins that override and dont yet have it